app/uk/gov/hmrc/hmrcemailrenderer/templates/childcare/childcareTaxFreeDevolved_B.scala.txt

```diff
@@ -7,10 +7,10 @@ Apply now for Tax-Free Childcare!
 
 You will be asked if you have been invited to apply. Please select 'no', and enter the date of birth of your youngest child.
 
-  Once you've applied for Tax-Free Childcare, you can log back into your account using the same link.  
+  Once you’ve applied for Tax-Free Childcare, you can log back into your account using the same link.  
 
 Find out more
- You can find more information on Tax-Free Childcare and check if you're eligible at: www.childcarechoices.gov.uk  Thank you for your interest in the childcare service.  Yours faithfully,
+ You can find more information on Tax-Free Childcare and check if you’re eligible at: www.childcarechoices.gov.uk  Thank you for your interest in the childcare service.  Yours faithfully,
```

Comment:
As you are changing the text, is it worth adding a space at the end of those sentences?
Just a thought.